### ISSUE TITLE
Assimp import throws StackoverflowException can't be caught by try/catch in .NET env

### ISSUE THREAD

**Issue by ytaous** [assimp/assimp-net#31]
Using .NET dll (from NuGet), from time to time we are getting exception that can't be caught by try/catch. StackoverflowException is one example - https://msdn.microsoft.com/en-us/library/system.stackoverflowexception(v=vs.110).aspx You can download the mode from http://static.tf3dm.com/models/c3dc953mnm68-Colloseum-by-William.zip to repro the issue. 

We also found other uncaughtable exception. Here is the details:

FatalExecutionEngineError occurred HResult=-2146233082 Message=Exception of type 'System.ExecutionEngineException' was thrown. InnerException: 

http://static.tf3dm.com/models/1/b6193a6p5m2o-Glass_Sparkling_Water.rar

Can all these exceptions be handled by assimp code? In our continuous process, it would stop when this kind of exception triggered. 
Thanks. 
-Ethan

**Comment by chuggafan** [comment]
StackOverflow errors can't be caught no matter what, if you can change the code where it's failing if you follow the call trees then you can fix it yourself, I for one would love to see this repo updated to the newest ASSIMP versions, but it seems like this project has been completely abandoned

**Comment by Starnick** [comment]
No, it's just that I don't use github (and also I've been busy with other projects):

https://bitbucket.org/Starnick/assimpnet

I'm visiting a bunch of defects and eventually I want to work towards updating to the latest native.
